src/libs/TransactionUtils.js

```diff
@@ -89,8 +89,14 @@ function hasReceipt(transaction) {
  * @param {Object} transaction
  * @returns {Boolean}
  */
-function areModifiedFieldsPopulated(transaction) {
-    return transaction.modifiedMerchant !== CONST.TRANSACTION.UNKNOWN_MERCHANT && transaction.modifiedAmount !== 0 && transaction.modifiedCreated !== '';
+function areRequiredFieldsEmpty(transaction) {
```

Comment:
We will have conflicts with this one https://github.com/Expensify/App/pull/26603 lets merge that first and sync iup this branch with it

Reply:
That one doesn't touch TransactionUtils 😕

Reply:
@mountiny @MariaHCD no conflicts 😁

Reply:
it adds the `PARTIAL_TRANSACTION_MERCHANT` thats what I meant, maybe there will be no conflict, but I just wanted to go in order 😅 been waiting hours for someone to merge it